plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.xml

```diff
@@ -15,5 +15,87 @@
     <keyword label="%appEngineNameWithSpace" 
              id="com.google.cloud.tools.eclipse.appengine.ui.AppSpaceEngineKeyword"/>
   </extension>
+  <extension
+        point="org.eclipse.ui.menus">
+     <menuContribution
+           allPopups="false"
+           locationURI="toolbar:org.eclipse.ui.main.toolbar?after=additions">
+        <toolbar
+              id="com.google.cloud.tools.eclipse.appengine.toolbar">
+           <command
+                 commandId="com.google.cloud.tools.eclipse.appengine.ui.showPopup"
+                 icon="icons/gae-16x16.png"
+                 id="com.google.cloud.tools.eclipse.appengine.actions"
+                 label="App Engine"
+                 style="pulldown"
+                 tooltip="App Engine">
+           </command>
+        </toolbar>
+     </menuContribution>
+       
+     <menuContribution
+           allPopups="false"
+           locationURI="menu:com.google.cloud.tools.eclipse.appengine.actions">
+        <menu
+              id="com.google.cloud.tools.eclipse.appengine.actions.new"
+              label="New...">
+           <command
+                 commandId="org.eclipse.ui.newWizard"
+                 label="Native App Engine Standard Project"
```

Comment:
In this case we do because the command is _New_ but we provide a parameter to open a specific new wizard.